Source/WebDriver/SessionHost.cpp

```diff
@@ -41,6 +41,11 @@ void SessionHost::inspectorDisconnected()
 
 long SessionHost::sendCommandToBackend(const String& command, RefPtr<JSON::Object>&& parameters, Function<void (CommandResponse&&)>&& responseHandler)
 {
+    if (!isConnected()) {
```

Comment:
So, this is not specific to the socket implementation, can you update the bug title and commit message, pelase?